glide-core/redis-rs/redis/tests/support/util.rs

```diff
@@ -125,3 +129,16 @@ pub async fn engine_version_less_than(min_version: &str) -> bool {
     }
     false
 }
+
+pub async fn kill_connections(client: &mut ClusterConnection) {
```

Comment:
```suggestion
pub async fn kill_all_connections(client: &mut ClusterConnection) {
```
also add short description